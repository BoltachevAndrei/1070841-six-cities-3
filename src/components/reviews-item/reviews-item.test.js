import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsItem from './reviews-item.jsx';

const review = {
  id: 203,
  user: {
    id: 1,
    name: `Test User`,
    avatar: `img/avatar.svg`,
    isPro: true
  },
  text: `Test review text`,
  rating: 4,
  date: new Date(`1995-06-24`)
};

it(`ReviewsItem renders correctly`, () => {
  const tree = renderer
    .create(<ReviewsItem
      id={review.id}
      review={review}
    />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
