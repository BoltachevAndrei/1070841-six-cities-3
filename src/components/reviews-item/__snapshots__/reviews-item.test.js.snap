// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsItem renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar.svg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Test User
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          4
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Test review text
    </p>
    <time
      className="reviews__time"
      dateTime="1995-06-24"
    >
      June 1995
    </time>
  </div>
</li>
`;
